README.md

```diff
@@ -82,6 +82,28 @@ To display the content of the rich editor field you must call the twig filter:
 
 You can see an example in the [test application](/tests/Application/templates/bundles/SyliusShopBundle/Product/Show/Tabs/Details/_description.html.twig)
 
+### Custom rendering of elements
+
+You can also render your elements with some custom DOM around that. To do so, you have access to a twig filter that 
+gives you the elements list :
+
+```twig
+{% set elements = content|monsieurbiz_richeditor_get_elements %}
+```
+
+And then you can either render them all : 
+
+```twig
+{{ monsieurbiz_richeditor_render_elements(elements) }}
+```
+
+Or one buy one :
+```twig
+{% for element in elements %}
+    {{ monsieurbiz_richeditor_render_element(element) }}
```

Comment:
Why you made a function instead of a filter ?

Reply:
For a simple reason : the filter name was already taken by already existing behavior, and I couldn't think about another clear name. Do you have any in mind ?

Reply:
Maybe the current `monsieurbiz_richeditor_render_element` filter have to be renamed `monsieurbiz_richeditor_render_field` ?

Reply:
Ok, I did not want to introduce any BC break. But that makes perfect sense

Reply:
@jacquesbh WDYTOT ?

Reply:
Erf. I agree with both of you. The name is not clear and a BC break is not ideal.

Let's do this since we are in RC, it's time to change it, otherwise it'll be too late.

Reply:
Maybe we can make a dedicated PR for this to give us some hours to update other plugins?

Reply:
And I realize that we need a changelog 😅